Catalyst.Fabric.Authorization.Client/AuthorizationClient.cs

```diff
@@ -3,38 +3,90 @@
 using System.Linq;
 using System.Net;
 using System.Net.Http;
+using System.Threading;
 using System.Threading.Tasks;
 using Catalyst.Fabric.Authorization.Client.Extensions;
 using Catalyst.Fabric.Authorization.Client.Routes;
 using Catalyst.Fabric.Authorization.Models;
 using Catalyst.Fabric.Authorization.Models.Requests;
 using Newtonsoft.Json;
+using Polly;
 
 namespace Catalyst.Fabric.Authorization.Client
 {
     public class AuthorizationClient
     {
-        
+        private Policy RetryPolicy;
+        private Policy CircuitBreakerPolicy;
+
+        private int MaxRetryAttempts;
+
+        private int TimeSpanInMinutes;
+
+        private int NumberOfErrorsBeforeCircuitBreak;
+
         private readonly HttpClient _client;
 
-        public AuthorizationClient(HttpClient client)
+        public AuthorizationClient(HttpClient client, bool useCircuitBreaker = false, bool useRetry = false)

```

Comment:
should we consolidate the constructor here and simply specify a boolean like `useConnectionResiliancy`? I would imaging if someone wants resiliancy, they would want both the circuit breaker and the retry policies.